README.md

```diff
@@ -1,8 +1,17 @@
-# Jellyfin.Plugins.Gotify
-Gotify Notification Agent for Jellyfin
+<h1 align="center">Jellyfin Gotify Plugin</h1>
+<h3 align="center">Part of the <a href="https://jellyfin.org/">Jellyfin Project</a></h3>
 
-Get started with Gotify at https://gotify.net/
+## About
+The Jellyfin Gotify plugin can be used for sending notifications to a hosted <a href="https://gotify.net/">Gotify server.</a>
```

Comment:
I would suggest a newline between any header and the next line.

Reply:
Should I do the same for everything? Doesn't change how it looks in markdown though